tests/data/rolling_stocks/electric_rolling_stock.json

```diff
@@ -977,5 +977,6 @@
     "BAPR",
     "TVM300",
     "TVM430"
-  ]
+  ],
+  "freight_compatible": false
```

Comment:
Why not make this one compatible?

Reply:
Mostly because this one is electric and I don't think there is any electric train for fret at the moment. Also because it felt it would be good to have different kind of values for test data. What do you thing?

Reply:
Alright, can't argue with physics.